tests: update pytest/pytest-django

blueyed opened the pull request.

blueyed commented:
Needs pytest-django 3.3.1, waiting for Travis to release it.

rpkilby commented:
How is this succeeding? The requirements specify pytest-django `3.3.2`, but the latest is `3.3.0`.

blueyed commented:
The latest should be 3.3.2.

blueyed commented:
(I just released it)

blueyed commented:
(you might need to clean your pip cache, since it might not look for a new version really for half an hour or so)

rpkilby commented:
Right - it's just confusing that this passed *before* `3.3.2` was released.

blueyed commented:
It used 3.3.2: https://travis-ci.org/encode/django-rest-framework/jobs/395156088#L466

It failed with 3.3.0 before, and then I've fixed and rebased/squashed it.

rpkilby commented:
I saw that, but when I checked PyPI, it only said the latest release was `3.3.0`. idk, maybe something was up with Warehouse.

¯\\\_(ツ)_/¯

blueyed commented:
Cache invalidation is hard.. ;)
